src/main/java/com/yo1000/sonarqube/collaboration/FizzBuzz.java

```diff
@@ -13,6 +13,12 @@ public String buzz(int value) {
     }
 
     public String fizzBuzz(int value) {
+        if (true) {
+            if (false) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS2583)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1066)